packages/documentation/src/stories/components/tooltip/tooltip.stories.ts

```diff
@@ -57,35 +57,50 @@ const meta: MetaComponent = {
       },
     },
     placement: {
-      name: 'Placement',
+      name: 'placement',
     },
     arrow: {
-      name: 'Arrow',
+      name: 'arrow',
       control: {
         type: 'boolean',
       },
     },
+    animation: {
+      options: ['none', 'pop-in'],
+    },
+    delay: {
+      name: 'delay',
+      description: 'Delay (in milliseconds) before the tooltip is shown.',
+      control: {
+        type: 'number',
+      },
+      table: {
+        category: 'props',
```

Comment:
As far as I saw, this is not a property of the `post-tooltip` component, but belongs to the `post-tooltip-trigger` component.
Therefore we should not mix it with the `post-tooltip` controls but instead add a second controls table for the trigger itself. Visit the accordion docs page to see what I mean.